manual/mapper/config/README.md

```diff
@@ -76,3 +78,95 @@ dependencies {
 ```
 
 You will find the generated files in `build/generated/sources/annotationProcessor`.
+
+### Lombok
+
+[Lombok](https://projectlombok.org/) is a popular library that automates boilerplate code, such as
+getters and setters. This can be convenient for mapped entities:
+
+```java
+import com.datastax.oss.driver.api.mapper.annotations.Entity;
+import com.datastax.oss.driver.api.mapper.annotations.PartitionKey;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@EqualsAndHashCode
+@ToString
+public class Product {
+  @PartitionKey @Getter @Setter private int id;
```

Comment:
If we are saying that ordering of annotation processors then maybe in this example we should follow the same ordering? so firstly Lombok annotations, and them ours:
```
@EqualsAndHashCode
@ToString
@Entity
public class Product {
 @Getter @Setter @PartitionKey private int id;
```

Reply:
The declaration order of the annotations does not matter, so I see no reason to imply that in our examples.